protocols/kad/src/behaviour.rs

```diff
@@ -371,12 +367,6 @@ impl Config {
         self
     }
 
-    /// Sets the amount of time to keep connections alive when they're idle.
-    pub fn set_connection_idle_timeout(&mut self, duration: Duration) -> &mut Self {
```

Comment:
Could you send a PR that deprecates this like we did in https://github.com/libp2p/rust-libp2p/pull/4648?

Reply:
open draft PR #4675, need confirm [discussion](https://github.com/libp2p/rust-libp2p/pull/4659#discussion_r1361468645]) below.